bower.json

```diff
@@ -15,7 +15,7 @@
     "polymer",
     "cosmoz"
   ],
-  "license": "Apache 2.0",
```

Comment:
I am not able to test demo page because it not able to find html and js files and if I try to use my fork repo to test I can do so it's say you need Apache 2.0 some license link that why I changed to a link but I revert back to older what we had